packages/main/src/Switch.js

```diff
@@ -100,6 +95,8 @@ const metadata = {
 		/**
 		 * Sets the accessible aria name of the component.
 		 *
+		 * <b>Note</b>: An accessible name reference needs to be set to an external label in all cases when <code>textOn</code> and <code>textOff</code> properties aren't set.
```

Comment:
Maybe here and for accessibleNameRef: 
* <b>Note</b>: We recommend that you set an <code>accessibleNameRef</code> to an external label or at least an <code>accessibleName</code>. Providing an <code>accessibleNameRef</code> or an <code>accessibleName</code> is mandatory in the cases when <code>textOn</code> and <code>textOff</code> properties aren't set.
*

Reply:
Done